backend/app/controllers/admin/exports_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Admin
+  class ExportsController < ApplicationController
+    def export
+      @csv_exporter = CsvExporter.new(survey: Survey.last)
```

Comment:
For future maybe take a param or route `/1` and then find the relevant Survey to export with Survey.find.